common/changes/@uifabric/set-version/api-extractor-fix_2018-11-01-18-43.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/set-version",
+      "comment": "Disable api-extractor build task",
+      "type": "patch"
```

Comment:
Build task edits should be none

Reply:
This was due to the change made in package.json to add api-extractor to the list of disabled tasks.

Reply:
Still should be none though. We don't need to publish for a change like this.